guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,14 +4,38 @@
 [id="katello-headline-features"]
 == Headline Features
 
-There are no highlights with Katello {KatelloVersion}.
+=== Content Access Modes Removed in Favor of Universal Simple Content Access
+Simple Content Access is now the only content access mode supported. When upgrading Katello, all existing organizations that use entitlement mode will migrate automatically to Simple Content Access. This change is not reversible. 
+
+Toggling SCA from the Organization Edit page, the Organization Create page, and the Organization List page is no longer available.
+
+=== Removed System Purpose-Related Host Statuses
+The following host statuses are removed:
+* Subscription Status (this column has also been removed from the All Hosts table)
+* System Purpose Status
+* SLA Status
+* Role Status
+* Usage Status
+* Addon Status
+
+=== Debian Content Support Expanded
+Import and export are now supported for repositories containing debian-type content. Aditionally, the host details page will now display more details for debian packages.
+
+=== Miscellaneous
+* Dropped dependencies on 'downshift', 'uglifier', 'minitest-tags', and 'simplecov'.
+* Katello has been updated to utilize Ruby 3.
+* Various UI tweaks, changes, and bugfixes.
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
 
-There are no upgrade warnings with Katello {KatelloVersion}.
+Simple Content Access is now the only supported content access mode. Existing organizations may be automatically migrated on Katello upgrade (see more info above). Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations, and will never have any rows. It will be removed in a future release or release candidate. Instead, use the 'Host - Installed Products' report. Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed, so you will experience errors if you try to use them. They will be removed in a future release or release candidate.
```

Comment:
```suggestion
Simple Content Access is now the only supported content access mode. Existing organizations will be automatically migrated on Katello upgrade (see more info above). Please note that the 'Subscription - Entitlements' report template does not work with SCA organizations, and will never have any rows. It will be removed in a future release or release candidate. Instead, use the 'Host - Installed Products' report. Additionally, please note that SCA-related API and Hammer endpoints and parameters have not yet been removed, so you will experience errors if you try to use them. They will be removed in a future release or release candidate.
```